server/src/recipe/index.js

```diff
@@ -1,8 +1,13 @@
-import { createRecipe, listPublishedRecipes, listUserRecipes } from './recipe.controller';
+import { createRecipe, deleteRecipe, listPublishedRecipes, listUserRecipes } from './recipe.controller';
+
 import { Router } from 'express';
 
 const router = Router();
 const path = '/recipes';
-router.get('/published', listPublishedRecipes).get('/userRecipes', listUserRecipes).post('/', createRecipe);
+router
+  .post('/delete', deleteRecipe)
```

Comment:
`.delete` and not `.post`

also, can you sort these so they are get -> post -> delete